### ISSUE TITLE
undefined local variable or method `role_changeable' under Passenger

### ISSUE THREAD

**Issue by AntonAL** [yhirano55/active_admin_role#16]
Using standard generated setup with `bin/rails generate active_admin_role:install`, I deployed my app on VDS with Nginx + Passenger.

Passenger cannot start the app and in `/var/log/nginx/error.log` I found following error:

> [ E 2018-04-11 20:05:03.0202 10865/T8 age/Cor/Con/CheckoutSession.cpp:285 ]: [Client 1-6772] Cannot checkout session because a spawning error occurred. The identifier of the error is 85af6ced. Please see earlier logs for details about the error.
> App 21291 stdout:
> App 21291 stdout: ReactOnRails: Set generated_assets_dir to default: public/webpack/production
> App 21291 stdout:
> [ E 2018-04-11 20:05:31.8733 10865/Tgq age/Cor/App/Implementation.cpp:304 ]: Could not spawn process for application /var/www/communitymoscow/current: An error occurred while starting up the preloader.
>  Error ID: 6c609093
>  Error details saved to: /tmp/passenger-error-VtzAFS.html
>  Message from application: undefined local variable or method `role_changeable' for #<ActiveAdmin::ResourceDSL:0x005556753024d0> (NameError)
>  /var/www/communitymoscow/releases/20180411195635/app/admin/admin_users.rb:3:in `block in <top (required)>'
>  /var/www/communitymoscow/shared/bundle/ruby/2.4.0/gems/activeadmin-1.2.1/lib/active_admin/dsl.rb:15:in `instance_exec'

**ActiveAdmin version**:

activeadmin (1.2.1)

**ActiveAdminRole version**:

active_admin_role (0.2.0)

**Rails version**:

gem 'rails', '~> 5.1.4'

**Ruby version**:

'2.4.1'

**Comment by yhirano55** [comment]
`role_changeable` is defined on `after_initialize`. Maybe you need to restart your rails server, I thought.

**Comment by AntonAL** [comment]
@yhirano55, It's ok on localhost. The problem is under Passenger in production.

**Comment by kvandake** [comment]
I get the same error. Can I resolve the problem?

**Comment by AntonAL** [comment]
@kvandake Not a solution, but workaround. I have deleted `role_changeable` from `app/admin/admin_users.rb` and explicitly declared abilities for each role in `app/models/ability.rb`. Just do it once and forget.

```ruby
class Ability
  include CanCan::Ability
  include ActiveAdminRole::CanCan::Ability

  def initialize(user)
    user ||= AdminUser.new

    if user.super_user?
      can :manage, :all
    elsif user.seo?
      can [:read, :update], [Area, Page, Event, Report, Preference]
      # ...
    else
      # Disable loading of abilities from DB
      #register_role_based_abilities(user)
    end

    # NOTE: Everyone can read the page of Permission Deny
    can :read, ActiveAdmin::Page, name: "Dashboard"
  end
```

**Comment by yhirano55** [comment]
I reproduced this problem on production env.

**Comment by yhirano55** [comment]
@AntonAL @kvandake Released v0.2.1. Please try latest version. If you face same error, please report to me.
